src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
Not completely, because main point of PR is to get rid of static source fields.
But they are kept, for those who want to load from same place without passing it each time to load & save.